include/nlohmann/detail/iterators/iteration_proxy.hpp

```diff
@@ -187,3 +223,8 @@ class tuple_element<N, ::nlohmann::detail::iteration_proxy_value<IteratorType >>
     #pragma clang diagnostic pop
 #endif
 } // namespace std
+
+#if JSON_HAS_RANGES
+    template <typename IteratorType>
+    inline constexpr bool ::std::ranges::enable_borrowed_range<::nlohmann::detail::iteration_proxy<IteratorType>> = true;
+#endif
```

Comment:
I assume modern MSVC can handle `<::nlohmann` now?

Reply:
I have no idea... MSVC is (to me) always good for a surprise...

Reply:
I think this was related to digraphs/trigraphs which have been removed in C++17? So it shouldn't be a concern in a C++20 code path. But as you say, it is MSVC we're talking about ... ;-)